int8 frameId

int16 point1x
int16 point1y
int16 point1z

int16 point2x
int16 point2y
int16 point2z

int16 point3x
int16 point3y
int16 point3z

int16 point4x
int16 point4y
int16 point4z

int16 point5x
int16 point5y
int16 point5z

int16 point6x
int16 point6y
int16 point6z

int16 point7x
int16 point7y
int16 point7z

int16 point8x
int16 point8y
int16 point8z


int16 point9x
int16 point9y
int16 point9z

int16 point10x
int16 point10y
int16 point10z

int16 point11x
int16 point11y
int16 point11z

int16 point12x
int16 point12y
int16 point12z

int16 point13x
int16 point13y
int16 point13z

int16 point14x
int16 point14y
int16 point14z

int16 point15x
int16 point15y
int16 point15z

int16 point16x
int16 point16y
int16 point16z